src/plugins/intel_gna/src/gna_plugin.hpp

```diff
@@ -27,16 +27,18 @@
 #include "gna_plugin_config.hpp"
 #include "log/debug.hpp"
 #include "log/log.hpp"
+#include "pre_post_process/transposition_info.hpp"
 
 namespace ov {
 namespace intel_gna {
 namespace request {
-
 class ModelWrapper;
 class WorkerPool;
 class Worker;
 }  // namespace request
 
+using namespace ov::intel_gna::pre_post_process;
```

Comment:
it is safer to not use using namespace in header

Reply:
fixed